test/rebalancer/bucket_ref.test.lua

```diff
@@ -20,8 +20,6 @@ _ = test_run:switch('box_2_a')
 vshard.storage.bucket_force_create(101, 100)
 _ = test_run:switch('box_1_a')
 
-box.space._bucket:replace{1, vshard.consts.BUCKET.ACTIVE}
-
```

Comment:
We've already created bucket 5 lines above and it's `ACTIVE`. No need to do that one more time, AFAICS.